backend/server/acl.go

```diff
@@ -188,6 +188,59 @@ func enforceWorkspaceDeveloperSheetRouteACL(plan api.PlanType, path string, meth
 	return nil
 }
 
+var issueStatusRegex = regexp.MustCompile(`^/issue/(?P<issueID>\d+)/status`)
+
+func enforceWorkspaceDeveloperIssueRouteACL(path string, method string, queryParams url.Values, principalID int, getIssueCreatorIDAndProjectID func(issueID int) (int, int, error), getProjectMemberIDs func(projectID int) ([]int, error)) *echo.HTTPError {
+	if !strings.HasPrefix(path, "/issue") {
+		return nil
+	}
+
+	if method == "GET" {
+		// For /issue route, users other than BytebaseBot cannot list other users' issues.
```

Comment:
Update the sentence: require the caller principal to be the same as the user in the query.